zebrad/src/commands/start/sync.rs

```diff
@@ -6,9 +6,24 @@ use tower::{Service, ServiceExt};
 use tracing_futures::Instrument;
 use zebra_chain::{block::BlockHeaderHash, types::BlockHeight};
 
+use zebra_network as zn;
+use zebra_state as zs;
+
+/// Get the heights of the blocks for constructing a block_locator list
+#[allow(dead_code)]
+pub fn block_locator_heights(tip_height: BlockHeight) -> impl Iterator<Item = BlockHeight> {
```

Comment:
i've been pretty intentional about ordering and keeping things ordered the same way rustdoc orders items so I'd prefer to keep functions after structs/impls enums and traits and type definitions after fns.

Reply:
Here's the ordering I've been aiming for

![Screenshot from 2020-06-23 08-57-15](https://user-images.githubusercontent.com/1993852/85426646-9e2c1180-b52f-11ea-8d54-5b0ded110bcb.png)

Reply:
Sure, happy to change it back, I moved it just because I was surprised to find extra types at the end of the file.

Reply:
Put them back in `06486c0`.